terraform/aws/ecs-cluster/iam.tf

```diff
@@ -43,7 +43,9 @@ resource "aws_iam_role_policy" "ecs_instance" {
         "ecr:GetDownloadUrlForLayer",
         "ecr:BatchGetImage",
         "ecs:StartTask",
-        "autoscaling:*"
+        "autoscaling:*",
+        "route53:ChangeResourceRecordSets",
```

Comment:
I think we should restrict this to only allow changes on the internal hosted zone

Reply:
How would we restrict it to only the internal hosted zone in aim roles?